test/test-sample.el

```diff
@@ -63,6 +60,15 @@
              (ert-fail "expected throw"))
            42)))
 
+(ert-deftest modtest-string ()
+  (should (string= "uname -a" (sample-mod-test-return-uname-cmd "-a"))))
+
+(ert-deftest modtest-uname ()
+  (let ((ref-uname-a-output (progn
+                              (require 'subr-x)
```

Comment:
I think you can move the "(require 'subr-x)" to at the top of file and can eliminate the progn.

Reply:
Ah, yes.  The progn was here because earlier I had the test from `modtest-string` inside `modtest-uname`. I'll do that tomorrow.

Reply:
I have made that change.